modules/mono/glue/GodotSharp/GodotSharp/Core/DelegateUtils.cs

```diff
@@ -76,6 +76,11 @@ private enum TargetKind : uint
 
         internal static bool TrySerializeDelegate(Delegate @delegate, Collections.Array serializedData)
         {
+			if (@delegate is null)
+			{
+				return false;
+			}
```

Comment:
Sorry for only noticing now... but I think C# code should be indented with 4 spaces and not tabs.

See `modules/mono/.editorconfig`, there should be a way for your IDE to pick it up and enforce this style properly.